packages/auth/src/grant/routes.test.ts

```diff
@@ -458,7 +458,7 @@ describe('Grant Routes', (): void => {
         expect(redirectSpy).toHaveBeenCalledWith(redirectUrl.toString())
         expect(ctx.session.nonce).toEqual(grant.interactNonce)
 
-        scope.isDone()
+        // scope.done() // TODO: fails
```

Comment:
This test _should_ work once `nock` is removed from the test above
```suggestion
        scope.done()
```